### PR TITLE
Updated Dropout operator to support Opset 12 and 13.

### PR DESCRIPTION
**Pull request by shahirdaya**
As of Opset 12, the Dropout operator now supports training mode. I have created tests for this. The test is an average of 20 runs of the operator. It checks that the ratio of dropouts in the output is almost equal to the  ratio requested.
Opset 13 supports a bfloat16 input tensor. I have verified that tf.nn.dropout works fine with bfloat16 tensors. I have not created a test case for this.
This will close issue #552

### CODE REVIEW

**Comment by chinhuang007** [comment]
A few onnx backend tests failed, as seen in CI. You can run `python test_onnx_backend.py -v OnnxBackendNodeModelTest.test_training_dropout_cpu` to see one of them. I am not sure the tests should produce the exact same results. Please take a look. Thanks!

**Comment by shahirdaya** [comment]
@chinhuang007 I am looking into this. Thanks

**Review by chinhuang007 — APPROVED** [review]
LGTM!
